hls4ml/writer/vivado_accelerator_writer.py

```diff
@@ -124,6 +131,23 @@ def write_axi_wrapper(self, model):
                     newline += indent + '#pragma HLS INTERFACE ap_ctrl_none port=return\n'
                     if model.config.get_config_value("IOType") == 'io_stream':
                         newline += indent + '#pragma HLS DATAFLOW\n'
+            elif '//hls-fpga-machine-learning insert load weights' in line:
```

Comment:
I would strongly discourage this style of code in the writer. What's the point of templates if they are blank and all the logic is written by the writer? Just clutters the writer. This is much better addressed with a template that contains compile-time constants to select the functionality and just fill out the few lines with specific weights

Reply:
Isn't this a copy mainly from `vivado_writer.py` lines 130-139? Do they both need to change?

Reply:
@jmitrevs, yes I moved the code from the `vivado_write.py` to the `vivado_accelerator_write.py`.